client/galaxy/scripts/mvc/form/form-input.js

```diff
@@ -109,9 +109,9 @@ define([], function() {
             tmp +=              '<div class="ui-form-info">';
             if (options.help) {
                 tmp +=              options.help;
-                if (options.argument && options.help.indexOf('(' + options.argument + ')') == -1) {
-                    tmp += ' (' + options.argument + ')';
-                }
+            }
+            if (options.argument && options.help.indexOf('(' + options.argument + ')') == -1) {
+                tmp +=              ' (' + options.argument + ')';
```

Comment:
I just looked at the line you changed, so you are probably right. Not my call anyway.

:+1: